src/main/java/CarRacing/RacingController.java

```diff
@@ -0,0 +1,32 @@
+package CarRacing;
+
+import CarRacing.domain.Competition;
+import CarRacing.view.InputView;
+import CarRacing.view.ResultView;
+
+public class RacingController {
+
+    private static final int DISTANCE_PER_TRY = 1;
+
+    public static void start() {
+        String[] nameOfCars = InputView.getCars();
+        int numberOfTry = InputView.getNumberOfTry();
+
+        Competition competition = new Competition();
+        try {
+            competition.entry(nameOfCars);
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+            throw new IllegalArgumentException(e.getMessage());
+        }
```

Comment:
```suggestion
        Competition competition = new Competition();
        competition.entry(nameOfCars);
```
예외를 잡아서 잡은 예외를 그대로 던지는 거면 try catch로 묶을 필요는 없어보여요